lib/matplotlib/tests/test_dates.py

```diff
@@ -465,6 +465,11 @@ def test_DayLocator():
    mdates.DayLocator(interval=1.0)
 
 
+def test_tz_utc():
+    dt = datetime.datetime(1970, 1, 1, tzinfo=mdates.UTC)
+    dt.tzname()
```

Comment:
I should add that my big complaint here is that this test, while exercising the method, does nothing to actually catch the original problem.